Modules/posixmodule.c

```diff
@@ -12768,6 +12773,12 @@ os_DirEntry_is_symlink_impl(DirEntry *self)
 #endif
 }
 
+static inline PyObject*
+DirEntry_get_module(DirEntry *self)
+{
+    return PyType_GetModule(Py_TYPE(self));
+}
+
```

Comment:
@vstinner Just to make sure you're aware, for future reviews: This only works because `DirEntry` is not subclassable.